### PR TITLE
Problem: sim tests are fail after #834

### PR DESCRIPTION
**Pull request by mmsqe**
Closes: https://github.com/crypto-org-chain/cronos/issues/921

👮🏻👮🏻👮🏻 !!!! REFERENCE THE PROBLEM YOUR ARE SOLVING IN THE PR TITLE AND DESCRIBE YOUR SOLUTION HERE !!!! DO NOT FORGET !!!! 👮🏻👮🏻👮🏻

# PR Checklist:

- [ ] Have you read the [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md)?
- [ ] Does your PR follow the [C4 patch requirements](https://rfc.zeromq.org/spec:42/C4/#23-patch-requirements)?
- [ ] Have you rebased your work on top of the latest master? 
- [ ] Have you checked your code compiles? (`make`)
- [ ] Have you included tests for any non-trivial functionality?
- [ ] Have you checked your code passes the unit tests? (`make test`)
- [ ] Have you checked your code formatting is correct? (`go fmt`)
- [ ] Have you checked your basic code style is fine? (`golangci-lint run`)
- [ ] If you added any dependencies, have you checked they do not contain any known vulnerabilities? (`go list -json -m all | nancy sleuth`)
- [ ] If your changes affect the client infrastructure, have you run the integration test?
- [ ] If your changes affect public APIs, does your PR follow the [C4 evolution of public contracts](https://rfc.zeromq.org/spec:42/C4/#26-evolution-of-public-contracts)?
- [ ] If your code changes public APIs, have you incremented the crate version numbers and documented your changes in the [CHANGELOG.md](https://github.com/crypto-org-chain/chain-main/blob/master/CHANGELOG.md)?
- [ ] If you are contributing for the first time, please read the agreement in [CONTRIBUTING.md](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md) now and add a comment to this pull request stating that your PR is in accordance with the [Developer's Certificate of Origin](https://github.com/crypto-org-chain/chain-main/blob/master/CONTRIBUTING.md#developer-certificate-of-originn).

Thank you for your code, it's appreciated! :)

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -545,13 +553,22 @@ func New(
...
+			),
+		)
+	} else {
+		app.StakingKeeper = *stakingKeeper.SetHooks(
+			stakingtypes.NewMultiStakingHooks(
+				app.DistrKeeper.Hooks(),
+				app.SlashingKeeper.Hooks(),
+			),
+		)
+	}
```
Comment (mmsqe) [inline app/app.go:571]:
```suggestion
	hooks := []stakingtypes.StakingHooks{
		app.DistrKeeper.Hooks(),
		app.SlashingKeeper.Hooks(),
	}
	if !skipGravity {
		hooks = append(hooks, app.GravityKeeper.Hooks())
	}
	app.StakingKeeper = *stakingKeeper.SetHooks(stakingtypes.NewMultiStakingHooks(hooks...))
```

**Review by thomas-nguy — COMMENTED** [review]
mmm this is the commit that add simulation to the gravity module if it helps
https://github.com/crypto-org-chain/gravity-bridge/pull/153

**Comment by mmsqe** [comment]
> mmm this is the commit that add simulation to the gravity module if it helps [crypto-org-chain/gravity-bridge#153](https://github.com/crypto-org-chain/gravity-bridge/pull/153)

it's call [NewGravityApp](https://github.com/adu-crypto/gravity-bridge/blob/4e3480977f05fc493273b93036fda240d1a0e112/module/app/sim_test.go#L293) with fauxMerkleModeOpt instead of [NewSimApp](https://github.com/crypto-org-chain/cronos/blob/3646c18ce32fca3c77095bb2af0f6f546dca2483/app/sim_test.go#L366)

**Review by yihuang — APPROVED** [review]
let's merge this to make sim test pass for now.

**Pull request merged**